websites.yml

```diff
@@ -46,3 +48,16 @@ username_patterns:
     invalid_patterns:
       - "\\.(com|net)"
       - "(\\.)\\1{1,}"  # consecutive '.' not allowed
+  openhub:
```

Comment:
Please re-read https://github.com/blackducksoftware/ohloh-ui/blob/master/app/models/concerns/account_validations.rb again, which I showed to you in Zulip ; there are more rules in there that you have missed. Also log into OpenHub, and change the text in your username setting to find some additional rules which can be added.  (You dont need to save your changes ; it will dynamically tell you about invalid usernames )

Reply:
sorry, I forgot to do it. but openhub doesn't dynamically tell me :(

Reply:
still not done.  Please re-read https://github.com/blackducksoftware/ohloh-ui/blob/master/app/models/concerns/account_validations.rb carefully

Reply:
still not done.  no guessing required.  Just read that code.

Reply:
Looks better now

Reply:
thanks.